### PR TITLE
New presence: Laftel

### PR DESCRIPTION
**Pull request by pikokr**
image
image
image

### CODE REVIEW

File: websites/L/Laftel/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+    "author": {
+      "name": "파랑이",
+      "id": "628595345798201355"
+    },
+    "contributors": [],
```
Comment (Bas950) [inline websites/L/Laftel/dist/metadata.json:6]: you dont need the contributors if you dont have it

File: websites/L/Laftel/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    "service": "Laftel",
+    "description": {
+      "en": "Anime streaming service in korea."
+    },
+    "url": [
+      "laftel.net"
+    ],
```
Comment (Bas950) [inline websites/L/Laftel/dist/metadata.json:13]: doesnt need to be in an array since its only 1

File: websites/L/Laftel/dist/metadata.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    "version": "0.0.1",
+    "logo": "https://asset.laftel.net/favicon.ico",
```
Comment (Bas950) [inline websites/L/Laftel/dist/metadata.json:15]: version 1.0.0
and upload to imgur

**Review by Bas950 — CHANGES_REQUESTED** [review]
Fix codacy issues

_btw if you are korean yourself please help us out translating PreMiD to korean_ :)

**Comment by doomlerd** [comment]
Provide proof screenshots.

File: websites/L/Laftel/dist/metadata.json [hunk 4]
```diff
@@ -0,0 +1,21 @@
+{
+    "author": {
+      "name": "파랑이",
+      "id": "628595345798201355"
+    },
+    "service": "Laftel",
+    "description": {
+      "en": "Anime streaming service in korea.",
+      "ko": "한국 애니메이션 스트리밍 서비스"
+    },
```
Comment (Bas950) [inline websites/L/Laftel/dist/metadata.json:10]: ko_KR

File: websites/L/Laftel/dist/metadata.json [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+    "url": "laftel.net",
+    "version": "1.0.0",
+    "logo": "https://i.imgur.com/XP7IP0A.png",
```
Comment (Bas950) [inline websites/L/Laftel/dist/metadata.json:13]: logo should atleast be 512x512 like the one in the presence

**Review by Bas950 — CHANGES_REQUESTED** [review]
Upload proof of it working

**Review by Bas950 — COMMENTED** [review]
Shouldn't you set a large image key in the presence?

**Comment by pikokr** [comment]
> Shouldn't you set a large image key in the presence?

I don't have images for it

**Comment by Bas950** [comment]
> > Shouldn't you set a large image key in the presence?
> 
> I don't have images for it

The logo of the website?

**Comment by Bas950** [comment]
And for small images keys you can download these and then upload them to the app. https://www.github.com/PreMiD/Presences/tree/master/.resources

**Comment by pikokr** [comment]
image
added small and large image keys.

**Review by Bas950 — APPROVED** [review]

**Review by Alanexei — APPROVED** [review]

**Pull request merged**